README.md

```diff
@@ -40,6 +40,10 @@ Our configuration is currently set to the following:
 - `Android Support Library = 27.1.0`
 - `Kotlin = 1.1+`
 
+This page should walk you through a generic set-up, but there are some gotchas for certain environments that we list on the following pages:
```

Comment:
We already have a "Cross platform frameworks" section in this README, which links to the sample code we built. Do you think this would fit better there?

Reply:
Ah, I forgot about that section

Yeah, perhaps something like changing it to be a list like:

* Xamarin - [sample app] / [documentation]
* React Native - [sample app] / [documentation]

?

Reply:
seems good to me 👍 no gotchas for xamarin yet tho

Reply:
Updated to be that :)